docs/content/docs/deployment/resource-providers/standalone/kubernetes.md

```diff
@@ -129,7 +129,7 @@ The *job artifacts* could be provided by these way：
 After creating [the common cluster components](#common-cluster-resource-definitions), use [the Application cluster specific resource definitions](#application-cluster-resource-definitions) to launch the cluster with the `kubectl` command:
 
 ```sh
-    $ kubectl create -f jobmanager-job.yaml
```

Comment:
You can search `jobmanager-job.yaml`, I still find 2 `jobmanager-job.yaml` in this page. They should be updated as well.

Chinese doc are similar.

Reply:
thank you. done